pkg/apis/serving/v1/configuration_defaults.go

```diff
@@ -67,5 +69,16 @@ func (cs *ConfigurationSpec) SetDefaults(ctx context.Context) {
 			return
 		}
 	}
+
+	configurationConfig := cconfig.FromContext(ctx)
+	apisConfig := config.Config{}
+	if configurationConfig != nil && configurationConfig.Defaults != nil {
+		apisConfig.Defaults = configurationConfig.Defaults.DeepCopy()
+	}
+	if configurationConfig != nil && configurationConfig.Features != nil {
+		apisConfig.Features = configurationConfig.Features.DeepCopy()
+	}
+	ctx = config.ToContext(ctx, &apisConfig)
+
```

Comment:
`configurationConfig := cconfig.FromContext(ctx)`

This will always be nil - the webhook has a config map watcher that updates the api's config store.

Reply:
I think there's a different fix here - 

`ResponseStartTimeoutSeconds` shouldn't realy be defaulted when it's empty cause it's implied that it's equal to `TimeoutSeconds`

https://github.com/knative/serving/blame/c3f2bfe4c93eec10be055e5f52047f709c0a8cc7/pkg/apis/serving/v1/revision_defaults.go#L66-L70

Reply:
ok - i see we perform defaulting when reconciling so it's reading the wrong store

Reply:
The fix should be to add the apiconfig.Store into the context using the same key that's in the apiconfig package

eg. here https://github.com/knative/serving/blob/c3f2bfe4c93eec10be055e5f52047f709c0a8cc7/pkg/reconciler/revision/config/store.go#L89

we should call

https://github.com/knative/serving/blob/c3f2bfe4c93eec10be055e5f52047f709c0a8cc7/pkg/apis/config/store.go#L69

Other stores in reconciler could have similar issues

Reply:
@dprotaso have you checked the description of the root cause here: https://github.com/knative/serving/issues/15616?

Reply:
As we discussed I will embed both configs.